### PR TITLE
Implemented Spacer Component

### PR DESCRIPTION
**Pull request by ruaridhm**

### CODE REVIEW

File: src/components/Spacer/Spacer.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  right: number;
+  bottom: number;
+  left: number;
+};
+
+const Spacer = ({ top, right, bottom, left }: Props) => {
+  top *= 8;
+  right *= 8;
+  bottom *= 8;
+  left *= 8;
```
Comment (MoSattler) [inline src/components/Spacer/Spacer.tsx:14]: Changing the props is [highly discouraged in react](https://reactjs.org/docs/components-and-props.html#props-are-read-only).

Comment (MoSattler) [reply src/components/Spacer/Spacer.tsx:14]: Try something like
```typescript
const topPadding = top * 8
```

Comment (limbovski) [reply src/components/Spacer/Spacer.tsx:14]: Good stuff Mo!

Comment (ruaridhm) [reply src/components/Spacer/Spacer.tsx:14]: Thanks for pointing this out Mo! I felt bad writing those lines at the time but I wasn't sure why. In my head a it was only props until it was passed into the component.

File: src/components/Spacer/Spacer.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const paddingTop = top * 8;
+  const paddingRight = right * 8;
+  const paddingBottom = bottom * 8;
+  const paddingLeft = left * 8;
+
+  return (
+    <div
+      style={{
+        padding: `${paddingTop}px ${paddingRight}px ${paddingBottom}px ${paddingLeft}px`,
+      }}
```
Comment (MoSattler) [inline src/components/Spacer/Spacer.tsx:20]: one last thing, can you make this styled-component styles?
```suggestion
      css={{
        padding: `${paddingTop}px ${paddingRight}px ${paddingBottom}px ${paddingLeft}px`,
      }}
```

also import 
```typescript
import 'styled-components/macro'
```

those two things do some CSS magic, like automatic [vendor prefixing](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix).

**Pull request merged**